dockstore-common/src/main/java/io/dockstore/common/SourceFileTest.java

```diff
@@ -0,0 +1,9 @@
+package io.dockstore.common;
+
+/**
+ * Contains tests specific to sourcefiles
+ * @author NatalieEO
+ */
+public class SourceFileTest {
```

Comment:
Looks like a category, but why is there a category being created?

Reply:
deleted